README.md

```diff
@@ -156,6 +156,19 @@ you may experience issues with saving files and hot module reload. Please review
 [this issue](https://github.com/shellscape/koa-webpack/issues/36#issuecomment-289565573)
 for more information and a workaround. 
 
+## Using with Server-Side-Render
+
+When set `serverSideRender` to true in `config.dev`, you can access `webpackStats` from `ctx.state.webpackStats`.
```

Comment:
"When `serverSideRender` is set to true in `config.dev`, `webpackStats` is accessible from `ctx.state.webpackStats`."